### DISCUSSION TITLE
Regrading Pakage Upgrade

### DISCUSSION THREAD

**Discussion by akcoder11** [termux/termux-packages#21282] [Q&A] [answered]
May I skip some packages which I don't want to upgrade while running this command "pkg upgrade" ?
please help me is there any way to stop upgrading package which I do not want to upgrade because I do not have much data .

**Comment by twaik** [comment]
We do not support partial upgrades.

**Reply by akcoder11** [reply]
thankyou very much for answering

**Comment by sylirre** [comment, marked as answer]
**Partial upgrades like upgrading few packages and skipping others are not supported.**

Termux is a rolling release distribution and it does not guarantee backwards compatibility between current package and older versions of dependencies. While Termux internally uses apt/dpkg package manager like Debian, the system maintenance workflows from the Debian are not applicable.

Final decision is up to you. However if you choose *partial upgrading*, don't file us bug reports how to fix the broken environment.

**Reply by akcoder11** [reply]
thank you very much for answering.
